app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
```

Comment:
Wait, just saw that `vehicle_id` is different than `agency`. Can you explain the difference?